c_glib/test/test-fixed-shape-tensor-data-type.rb

```diff
@@ -56,6 +60,7 @@ def test_nil_permutation
                                                     nil,
                                                     ["x", "y"])
     assert_equal([], data_type.permutation)
+    assert_equal([32, 8], data_type.strides)
```

Comment:
Could you use one `assert_equal` for easy to debug?

See also: https://www.clear-code.com/blog/2011/2/28.html (Japanese)

```suggestion
    assert_equal([[], [32, 8]],
                 [data_type.permutation, data_type.strides])
```